modules/c/nrt/source/Utils.c

```diff
@@ -295,30 +307,44 @@ NRTAPI(void) nrt_Utils_decimalToGeographic(double decimal, int *degrees,
     *minutes = (int) remainder;
     *seconds = fabs(remainder - (double) *minutes) * 60.0;
 
+    if (*degrees == 0)
+    {
+        if (*minutes == 0)
+        {
+            *seconds *= -1;
+        }
+        else
+        {
+            *minutes *= -1;
+        }
+    }
 }
 
 NRTAPI(double) nrt_Utils_geographicToDecimal(int degrees, int minutes,
                                              double seconds)
 {
     double decimal = fabs((double)degrees);
-    decimal += ((double) minutes / 60.0);
-    decimal += (seconds / 3600.0);
+    decimal += fabs(((double) minutes / 60.0));
```

Comment:
`(double)` cast not needed (I know it was here already) and don't need extra set of parens